### ISSUE TITLE
Detect shelling out to unzip to uncompress files

### ISSUE THREAD

**Issue by tas50** [chef/cookstyle#525]
### What category of cop is this?:

Please try to select as few as possible.

- [ ] ChefDeprecations - A cop that alerts on a deprecation in the Chef Infra Client
- [ ] ChefCorrectness - A cop that alerts when incorrect coding behavior is being used
- [ ] ChefSharing - A cop that alerts to missing functionality necessary for sharing cookbooks with others
- [ ] ChefStyle - A cop that alerts to a style best practice
- [ ] ChefModernize - A cop that alerts when a cookbook can be simplified or modernized with new functionality
- [ ] ChefEffortless - A cop that alerts on code that must be resolved to move to the Effortless pattern

### Describe the new cop:
We should detect shelling out to unzip to decompress files

### What it would trigger on:

```ruby
execute 'unzip 1password' do
  command "unzip #{Chef::Config[:file_cache_path]}/1Password-#{node['one_password']['version']}.zip"
  cwd '/Applications'
  not_if { ::File.directory?("/Applications/#{node['one_password']['appname']}") }
end
```
